index.js

```diff
@@ -38,153 +39,178 @@ module.exports.s3 = function(config, key, args, callback) {
 	);
 };
 
+var getDimArray = function( dims ) {
+	var dimArr = ( typeof dims === 'string'
+		? dims.split(',')
+		: dims
+	);
```

Comment:
Can this just be a one-liner now? Seems strange to split out if we don't need the trailing `.map`. Also, the paren isn't needed if we do that.